conda.bzl

```diff
@@ -121,5 +152,9 @@ load_conda_rule = repository_rule(
             default = EXECUTE_TIMEOUT,
             doc = "Timeout in seconds for each execute action",
         ),
+        "installer": attr.string(
+            default = "miniconda",
+            doc = 'Installer to use, either "miniconda" or "miniforge". Note that miniconda and miniforge have different OS/arch support.',
+        ),
```

Comment:
Please move it to be the first optional argument

Reply:
Done.